sdks/java/core/src/test/java/org/apache/beam/sdk/testing/FileChecksumMatcherTest.java

```diff
@@ -123,7 +122,7 @@ public void testMatcherThatVerifiesFileWithEmptyContent() throws IOException {
     FileChecksumMatcher matcher =
         new FileChecksumMatcher(
             "da39a3ee5e6b4b0d3255bfef95601890afd80709",
-            IOChannelUtils.resolve(tmpFolder.getRoot().getPath(), "*"));
+            tmpFolder.getRoot().toPath().resolve("*").toString());
```

Comment:
I believe we ran into issues with toPath().resolve("*") in Windows OS in the path.